src/gui/guiutility.cpp

```diff
@@ -23,12 +23,24 @@
 
 #include "common/asserts.h"
 
+namespace {
+    const QStringList whitelistedUrlSchemes = {
```

Comment:
Avoid global variables in general and global static non-POD types in particular. It is probably fine to have it as a local const in openBrowser, it's not on any hot path really.